models/Arena.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require('mongoose');
+const {
+  Schema
+} = mongoose;
+const MediaSchema = require('./schema/Media');
+const ArenaSchema = new Schema({
+  _event: {
+    type: Schema.ObjectId,
```

Comment:
rather typing again and again, we can define ObjectId at the top
`const ObjectId = Schema.ObjectId;`

Reply:
Nice idea, even better we can use the new ES6 destructuring feature 
`const { ObjectId } = Schema`

Reply:
Done in #22

Reply:
Great